New Build from Master on Events Changes

vermouth22 opened the pull request.

yann300 committed:
Merge pull request #16 from ethereum/david-profile-update

David's Profile Update

Va54 committed:
Add remix-website new version.

Va54 committed:
Fix wrong npm command.

Va54 committed:
Fix warnings on build stage

Va54 committed:
Remove Terms and Privacy anchors.

Va54 committed:
Remove aws deploy steps.

Va54 committed:
Add editorconfig.

araratpoghosyan committed:
Fix github pages deployment

yann300 committed:
Merge pull request #32 from ethereum/new-remix-website

Remix-website React version

Va54 committed:
Fix update homepage.

yann300 committed:
Update README.md

yann300 committed:
Merge pull request #34 from ethereum/fix/domain

Fix domain

yann300 committed:
remove homepage

Va54 committed:
Add file structure and data entries.

maksimbar committed:
Change grid behaviour on md breakpoint

yann300 committed:
Merge pull request #35 from ethereum/update/readme

Update Readme

LianaHus committed:
Merge pull request #36 from tmneth/master

Improve responsiveness on smaller screens

kuzdogan committed:
Fix old GitHub repo link in Footer

LianaHus committed:
Merge pull request #41 from kuzdogan/patch-1

Fix old GitHub repo link in Footer

LianaHus committed:
Update Learn section in Navbar.jsx

Documents -> Documentation

ryestew committed:
Merge pull request #43 from ethereum/UpdateLearnDocs

Update Learn section in Navbar.jsx

vermouth22 committed:
Update About.jsx

Fixed a wrong word in text block.

vermouth22 committed:
Update RemixIde.jsx

Fixed typos

vermouth22 committed:
Update Rewards.jsx

Updated text

ryestew committed:
Merge pull request #48 from ethereum/vermouth22-patch-4

Update Rewards.jsx

ryestew committed:
Merge pull request #47 from ethereum/vermouth22-patch-2

Update RemixIde.jsx

ryestew committed:
Merge pull request #46 from ethereum/vermouth22-patch-1

Update About.jsx

drafish committed:
i18n and chinese translations

vermouth22 committed:
Update Events.jsx

Updated Events

vermouth22 committed:
Add files via upload

vermouth22 committed:
Update Team.jsx

Change "Gitter Channels" to "Discord"

vermouth22 committed:
Merge pull request #55 from ethereum/vermouth22-patch-8

Update Team.jsx

vermouth22 committed:
Update Events.jsx

vermouth22 committed:
Merge pull request #54 from ethereum/vermouth22-patch-7

Update Events.jsx

LianaHus committed:
add URLs to events.jsx

LianaHus committed:
added UML parameter to the EventCard.jsx

LianaHus committed:
Title is a link as well EventCard.jsx

LianaHus committed:
Merge pull request #56 from ethereum/LianaHus-patch-1

add URLs to events.jsx

LianaHus committed:
Update Events.jsx

LianaHus committed:
Update EventCard.jsx

LianaHus committed:
Update Footer to add discord

LianaHus committed:
Add files via upload

LianaHus committed:
Merge pull request #58 from ethereum/LianaHus-patch-3

Update Footer to add discord

LianaHus committed:
Update discord-icon.svg

LianaHus committed:
Update Footer.jsx

LianaHus committed:
Updated color of discord-icon.svg

LianaHus committed:
rename SSA plugin

LianaHus committed:
Merge pull request #59 from ethereum/LianaHus-patch-4

rename SSA plugin

LianaHus committed:
Update EventCard.jsx

vermouth22 committed:
Merge pull request #60 from ethereum/LianaHus-patch-5

Update EventCard.jsx

LianaHus committed:
Update EventCard.jsx

vermouth22 committed:
Update Events.jsx

vermouth22 committed:
Add files via upload

vermouth22 committed:
Merge pull request #62 from ethereum/vermouth22-patch-8

Update Events.jsx

vermouth22 committed:
Update Events.jsx

LianaHus commented:
IMHO you are doing something wrong, you shouldn't have those changes. let's talk

LianaHus committed:
Update video to be a link to ide page

LianaHus committed:
Update About.jsx

LianaHus committed:
Update Events.jsx fixed the typo

LianaHus committed:
Merge pull request #67 from ethereum/LianaHus-patch-7

Update Events.jsx fixed the typo

joeizang committed:
change sequencing of eventsData array. fix key prop error in Team.jsx component. add site data json

joeizang committed:
update events with EthIndia details

joeizang committed:
updated image

wackerow committed:
fix: always render current year for copyright

wackerow committed:
chore: import Helvetica

wackerow committed:
feat: clamp header width

wackerow committed:
chore: rename ide section to start

wackerow committed:
feat: extract SubNavbar from Navbar

wackerow committed:
refactor: use URL constants

temporarily use dev links

wackerow committed:
chore: pixel details

wackerow committed:
feat: style sub nav

wackerow committed:
chore: adjust Navbar styling

wackerow committed:
build: install dev @svgr/webpack

wackerow committed:
feat: polish learn dropdown

wackerow committed:
feat: add ThemeDropdown component

update assets

wackerow committed:
chore: patch navbar bg color

wackerow committed:
feat: add theme color tokens

wackerow committed:
feat: add/implement useColorMode hook

wackerow committed:
refactor: update SVG assets

wackerow committed:
feat: implement new color theme tokens

wackerow committed:
chore: update svgs to use currentcolor

wackerow committed:
chore: update useColorMode

wackerow committed:
feat: update svgs and colors to use theming

wackerow committed:
chore: update svgs for new color theming

wackerow committed:
fix: desktop nav styling adjustment

wackerow committed:
chore: adjustt ThemeDropdown styling

wackerow committed:
fix: links, pass color mode to docs

wackerow committed:
fix: update useColorMode

wackerow committed:
feat: create useColorContinuity hook and url utils

wackerow committed:
feat: implement color continuity

wackerow committed:
chore: add key, manifest asset cleanup

wackerow committed:
refactor: navbar spacing

wackerow committed:
fix: check localstorage color mode on load

wackerow committed:
fix: color mode handling

refactor to use one instance of useColorMode inside App; pass state props. Fixes inconsistencies in colorMode values depending on component

wackerow committed:
fix: overflowing h1 on mobile

wackerow committed:
feat: style mobile menu

wackerow committed:
feat: always show theme dropdown component

wackerow committed:
Merge branch 'unification' into unification-i18n

wackerow committed:
chore: intl merger

wackerow committed:
refactor: extract useIntl hook

wackerow committed:
refactor: pass argument to removeParam

wackerow committed:
feat: create DocsLink component

related to continuity between project home and RTD docs by preparing "docs" links with the appropriate language and color mode currently being used.

wackerow committed:
feat: implement DocsLink component

deprecate getDocsHref util

wackerow committed:
chore: styling adjustments

wackerow committed:
feat: update subnavbar and other intl strings

wackerow committed:
chore: update remaining strings

wackerow committed:
feat: improve initial parsing of intl choice

include check for RTD codes and browser preference
fix: ignore case on lang param from docs

wackerow committed:
fix: mobile menu learn background

joeizang committed:
clean up Eventcard add Eventdata to localstorage

wackerow committed:
chore: update color mode icons and names

light|dark|black, sun|sunset|moon icons

wackerow committed:
chore: clean up header spacing

joeizang committed:
fix broken url for events

joeizang committed:
Merge pull request #75 from ethereum/joe-patch-01

Joe patch 01

wackerow committed:
fix: subnav hover style

wackerow committed:
chore: apply adjustments from team review

wackerow committed:
chore: align dropdown menu positioning

wackerow committed:
fix: use hover color

wackerow committed:
chore: clean up intl string

utilize `uppercase` styling where possible to limit translation issues with capitalization

wackerow committed:
Merge branch 'master' into unification-i18n

wackerow committed:
patch: remove duplicate key

wackerow committed:
chore: patch alignment

wackerow committed:
fix: intl string id order

wackerow committed:
chore: linting

wackerow committed:
feat: add Spanish, French, Russian locales

wackerow committed:
fix: external arrow layout

wackerow committed:
chore: import languages

wackerow committed:
fix: tablet sizing

wackerow committed:
chore: shorten Russian "docs" string

wackerow committed:
revert: dev links; use production

wackerow committed:
chore: adjust subnav border

ryestew committed:
Merge pull request #71 from wackerow/unification-i18n

Remix Website design revamp and unification

ryestew committed:
Merge branch 'master' into copyright-year

ryestew committed:
Merge pull request #70 from wackerow/copyright-year

fix: always render current year for copyright

wackerow committed:
fix: update remaining color tokens

ryestew committed:
Merge pull request #78 from wackerow/patch-color

Chore: use responsive color tokens

LianaHus committed:
rm vscode info

LianaHus committed:
updated the remix desktop link

LianaHus committed:
Update twitter-icon.svg

LianaHus committed:
Update twitter-icon.svg

LianaHus committed:
desktop url in footer updated

LianaHus committed:
fixing colors

LianaHus committed:
fix

LianaHus committed:
Update twitter-icon.svg

LianaHus committed:
Update twitter-icon.svg

joeizang committed:
Update discord Icon url

joeizang committed:
Merge pull request #85 from ethereum/joeizang-footer-patch

Update discord Icon url

yann300 committed:
dashboard

yann300 committed:
fix github action

yann300 committed:
Merge pull request #90 from ethereum/add_dashboard

dashboard
